src/sql/workbench/services/contextualization/common/serverContextualizationService.ts

```diff
@@ -120,6 +142,8 @@ export class ServerContextualizationService extends Disposable implements IServe
 	 */
 	private async sendServerContextualizationToCopilot(serverContext: string | undefined): Promise<void> {
 		if (serverContext) {
+			this._logService.info('Sending server contextualization to Copilot');
```

Comment:
Is `info` the correct log level, or should I be using `debug` level?

Reply:
I'm not sure since I see both being used in many places. @Charles-Gagnon are there any best practices here on when to use `info` vs `debug` for logging?